src/kOS.Safe/Encapsulation/ISerializableValue.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    public interface ISerializableValue
```

Comment:
Im not saying we should use the built in serializer framework. just the attribute. When i see an empty Interface it makes me want to make it an attribute. we could make a `kOSSerializable` if we want.